deploy-jenkins/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
 
 -
   name: "Deploy the Jenkins Instance"
-  command: "oc new-app --namespace {{ project_name }} -f {{ buildfarm_templates_dir }}/jenkins-persistent-template.json -p ENABLE_OAUTH={{ enable_oauth }}"
+  command: "oc new-app --namespace {{ project_name }} -f {{ buildfarm_templates_dir }}/jenkins-persistent-template.json -p ENABLE_OAUTH={{ enable_oauth }} -p MEMORY_LIMIT={{ master_memory_limit }} -p VOLUME_CAPACITY={{ master_volume_capacity }}"
```

Comment:
Instead of passing the variables as a parameter it can be dynamically added to the jinja template. There is an example [here](https://github.com/aerogear/digger-installer/blob/master/android-sdk/templates/android-sdk-persistent-template.j2#L236)